coldfront/core/project/tests/test_commands/test_get_survey_responses.py

```diff
@@ -0,0 +1,121 @@
+from django.test import TestCase
+from coldfront.core.project.models import *
+from django.contrib.auth.models import User
+from django.core.management import call_command
+
+
+from coldfront.core.project.models import (SavioProjectAllocationRequest,
+                                           Project,
+                                           ProjectAllocationRequestStatusChoice,
+                                           ProjectStatusChoice,
+                                           User)
+
+from io import StringIO
+import sys
+import json
+import os
+from csv import DictReader
+
+
+class TestGetSurveyResponses(TestCase):
+    """
+    Base Class for testing
+    """
+
+    def setUp(self):
+        # run setup commands
+        out, err = StringIO(), StringIO()
+        commands = [
+            'add_default_project_choices',
+            'import_field_of_science_data'
+        ]
+        sys.stdout = open(os.devnull, 'w')
+        for command in commands:
+            call_command(command, stdout=out, stderr=err)
+        sys.stdout = sys.__stdout__
+
+        # create dummy survey responses
+        fixtures = []
+        filtered_fixtures = []
+
+        # dummy params
+        allocation_type = SavioProjectAllocationRequest.FCA
+        pool = False
+
+        for index in range(5):
+            pi = User.objects.create(
+                username=f'test_user{index}', first_name='Test', last_name='User',
+                is_superuser=True)
+
+            project_status = ProjectStatusChoice.objects.get(name='Active')
+            allocation_status = ProjectAllocationRequestStatusChoice.objects.get(
+                name='Under Review')
+
+            project_prefix = 'fc_' if index % 2 else ''
+            project = Project.objects.create(name=f'{project_prefix}test_project{index}',
+                                             status=project_status)
+
+            survey_answers = {'a': f'answera_{index}', 'b': f'answerb_{index}',
+                              'roject_name': project.name,
```

Comment:
Looks like a typo.

Correct me if I'm wrong, but these keys aren't in the form, so you should be able to omit them, since they're going to be ignored anyway.

(We're just adding them when dumping the answers in the command.)

Reply:
Also, I think `a` and `b` here correspond to `scope_and_intent` and `computational_aspects`. We're not creating the request via the form, so this works, but it may be more clear to use the actual keys.

Reply:
> Looks like a typo.
> 
> Correct me if I'm wrong, but these keys (`project_name` and `project_typo`) don't need to be here, since they're going to be added by the command.

since the command outputs with these two fields, I need to include them in the fixture to pass the tests.

Reply:
If we add these two keys within the `survey_answers` field directly, we're not testing that the management command adds them, because they're already there.

Also, these keys wouldn't be in the `survey_answers` in practice; the field would only contain the fields from `SavioProjectSurveyForm`.